24_clustering.Rmd

```diff
@@ -17,9 +18,122 @@ features/samples that share pattern.  For example, with clustering, we
 can find group of samples that share similar community
 composition. There are multiple clustering algorithms available.
 
+## Introduction to bluster package
+
+*bluster* is a Bioconductor package specialized in clustering. It offers 
+multiple algorithms such as hierarchical clustering, DBSCAN, K-means, 
+amongst others. The first thing to do when using this package is to load 
+it. It's also possible to use `transformCounts` to transform the data 
+for the clustering.
```

Comment:
I would not explain `transformCounts` here. It is a standard operation that can be done with any method. The tutorial will have massive redundancy if we repeat this same message with various different methods. Standard data manipulation is already covered in its own chapter, in these examples we focus on our specific methodology. Transformations etc. can be discussed if there is something very specific related to the given methodology.